packages/wrangler/src/api/dev.ts

```diff
@@ -189,7 +189,7 @@ export async function unstable_dev(
 		localProtocol: options?.localProtocol,
 		httpsKeyPath: options?.httpsKeyPath,
 		httpsCertPath: options?.httpsCertPath,
-		experimentalAssets: options?.experimentalAssets,
+		experimentalAssets: undefined,
```

Comment:
Are we tracking unstable_dev support elsewhere? If not, worth leaving a TODO?

Reply:
we talked about this in our sync meeting yesterday, and we agreed that there's not much value to add support in `unstable_dev` since:

- the long-term plan is to move away from it
- `unstable_dev` is only used by Pages and fixture tests

But if you have other thoughts happy to hear them

Reply:
startWorker will supersede unstable_dev and looks to be considered in this PR – no need to add new features to unstable_dev